### DISCUSSION TITLE
Taxonomy Committee Meeting Reminder

### DISCUSSION THREAD

**Discussion by Jegelewicz** [ArctosDB/arctos#6273] [Announcements]
@ArctosDB/taxonomy a reminder that we have a meeting tomorrow (May 11) at 10AM PDT in the Arctos Zoom Room. Hope to see some of you there!

[Taxonomy GitHub Project](https://github.com/orgs/ArctosDB/projects/27)
[Rolling Agenda](https://docs.google.com/document/d/1gxMD9uzsyGl2zKaMitQ_6tH33G9RUoVpwDWzuB2Xxmw/edit#)

**Comment by Jegelewicz** [comment]
@ArctosDB/taxonomy a reminder that we have a meeting July 13 at 10AM PDT in the Arctos Zoom Room. Hope to see some of you there!

[Taxonomy GitHub Project](https://github.com/orgs/ArctosDB/projects/27)
[Rolling Agenda](https://docs.google.com/document/d/1gxMD9uzsyGl2zKaMitQ_6tH33G9RUoVpwDWzuB2Xxmw/edit#)

**Reply by genevieve-anderegg** [reply]
Hello! I totally missed this, accidentally deleted it from my calendar. My apologies Teresa! There are definitely some items that Phyllis and I wish to discuss; would we be able to reschedule this for sometime soon? Or have an additional meeting?
Sincerest apologies!!

**Reply by Jegelewicz** [reply]
Next Tuesday or Wednesday?

**Reply by genevieve-anderegg** [reply]
I am free all next Tuesday and after 11AM MST on Wednesday! @sharpphyl @ArctosDB/taxonomy ?
Thank you Teresa!

**Reply by Jegelewicz** [reply]
Added to the calendar for 1PM

**Comment by Jegelewicz** [comment]
@ArctosDB/taxonomy a reminder that we have a meeting November 9 at 10AM PDT in the Arctos Zoom Room. Hope to see some of you there!

[Taxonomy GitHub Project](https://github.com/orgs/ArctosDB/projects/27)
[Rolling Agenda](https://docs.google.com/document/d/1gxMD9uzsyGl2zKaMitQ_6tH33G9RUoVpwDWzuB2Xxmw/edit#)

**Comment by Jegelewicz** [comment]
@ArctosDB/taxonomy a reminder that we have a meeting this Thursday, January 11 at 10AM PDT in the Arctos Zoom Room. Hope to see some of you there!

[Taxonomy GitHub Project](https://github.com/orgs/ArctosDB/projects/27)
[Rolling Agenda](https://docs.google.com/document/d/1gxMD9uzsyGl2zKaMitQ_6tH33G9RUoVpwDWzuB2Xxmw/edit#)

**Comment by camwebb** [comment]
@Jegelewicz Is there a meeting on 3/14 as per the GoogleDoc? Bunch of things I'd like to discuss.

**Reply by Jegelewicz** [reply]
Yes, there is and this can be the reminder!

**Comment by Jegelewicz** [comment]
@ArctosDB/taxonomy a reminder that we have a meeting tomorrow (March 14) at 10AM PDT in the Arctos Zoom Room. Hope to see some of you there!

[Taxonomy GitHub Project](https://github.com/orgs/ArctosDB/projects/27)
[Rolling Agenda](https://docs.google.com/document/d/1gxMD9uzsyGl2zKaMitQ_6tH33G9RUoVpwDWzuB2Xxmw/edit#)

**Comment by genevieve-anderegg** [comment]
Hi @Jegelewicz ! I have another meeting directly on top of Taxonomy tomorrow (May 8th) so I won't be able to make it. DMNS has no pressing taxonomy concerns. See you next time!

**Reply by Jegelewicz** [reply]
But Taxonomy is May 9 - maybe you are looking at Agents?

**Reply by genevieve-anderegg** [reply]
Ope I just typed the date completely wrong, but I actually am busy during Taxonomy on May 9th, Thursday!

**Comment by Jegelewicz** [comment]
Taxonomy Committee Member,

We have a standing meeting of the Arctos Taxonomy Committee on the calendar tomorrow at 10AM PDT in the Arctos Zoom Room.

The [agenda](https://docs.google.com/document/d/1gxMD9uzsyGl2zKaMitQ_6tH33G9RUoVpwDWzuB2Xxmw/edit#heading=h.zdqph5yfhyw) is currently empty. If nothing is added before the meeting time tomorrow, I will cancel the meeting for this month.

**Comment by genevieve-anderegg** [comment]
@ArctosDB/taxonomy we have a meeting today (September 12) at 10AM PDT in the Arctos Zoom Room. I am leading the committee now, hope to see some of you there to talk about any taxonomy discussion items! 
[Taxonomy Github Project ](https://github.com/orgs/ArctosDB/projects/27)
[Taxonomy Agenda](https://docs.google.com/document/d/1gxMD9uzsyGl2zKaMitQ_6tH33G9RUoVpwDWzuB2Xxmw/edit)

**Comment by genevieve-anderegg** [comment]
@ArctosDB/taxonomy we have a meeting today (January 9) at 10AM PDT in the Arctos Zoom Room. We don't have any big discussion items, if you have something you wish to discuss please put it on the agenda!
[Taxonomy Github Project](https://github.com/orgs/ArctosDB/projects/27)
[Taxonomy Agenda](https://docs.google.com/document/d/1gxMD9uzsyGl2zKaMitQ_6tH33G9RUoVpwDWzuB2Xxmw/edit)

**Comment by genevieve-anderegg** [comment]
@ArctosDB/taxonomy Taxonomy committee meeting have a meeting on Thursday March 13th at 10AM PDT in the Arctos Zoom Room. We don't have any big discussion items, if you have something you wish to discuss please put it on the agenda!
[Taxonomy Github Project](https://github.com/orgs/ArctosDB/projects/27)
[Taxonomy Agenda](https://docs.google.com/document/d/1gxMD9uzsyGl2zKaMitQ_6tH33G9RUoVpwDWzuB2Xxmw/edit)

**Comment by genevieve-anderegg** [comment]
@ArctosDB/taxonomy Taxonomy committee today! Thursday May 8th at 10AM PDT in the Arctos Zoom Room. We don't have any big discussion items, if you have something you wish to discuss please put it on the agenda!
https://github.com/orgs/ArctosDB/projects/27
[Taxonomy Agenda](https://docs.google.com/document/d/1gxMD9uzsyGl2zKaMitQ_6tH33G9RUoVpwDWzuB2Xxmw/edit)

**Reply by camwebb** [reply]
Sorry to miss this one. I'll try to be at the July 10 meeting.

**Comment by genevieve-anderegg** [comment]
@ArctosDB/taxonomy Taxonomy committee tomorrow! Thursday July 10 at 10AM PDT in the Arctos Zoom Room. We don't have any big discussion items, if you have something you wish to discuss please put it on the agenda!
https://github.com/orgs/ArctosDB/projects/27
[Taxonomy Agenda](https://docs.google.com/document/d/1gxMD9uzsyGl2zKaMitQ_6tH33G9RUoVpwDWzuB2Xxmw/edit)

**Comment by genevieve-anderegg** [comment]
@ArctosDB/taxonomy Taxonomy committee tomorrow! Thursday September 111 at 10AM PST/ 11 MST/ 1PM EST in the Arctos Zoom Room. We don't have any big discussion items, if you have something you wish to discuss please put it on the agenda!
https://github.com/orgs/ArctosDB/projects/27
[Taxonomy Agenda](https://docs.google.com/document/d/1gxMD9uzsyGl2zKaMitQ_6tH33G9RUoVpwDWzuB2Xxmw/edit)

**Comment by genevieve-anderegg** [comment]
@ArctosDB/taxonomy Taxonomy committee meeting TODAY! Thursday November 13 at 10AM PST/ 11 MST/ 1PM EST in the Arctos Zoom Room. We don't have any big discussion items, if you have something you wish to discuss please put it on the agenda!
https://github.com/orgs/ArctosDB/projects/27
[Taxonomy Agenda](https://docs.google.com/document/d/1gxMD9uzsyGl2zKaMitQ_6tH33G9RUoVpwDWzuB2Xxmw/edit)

**Comment by genevieve-anderegg** [comment]
@ArctosDB/taxonomy @dustymc  Taxonomy committee meeting TODAY! Thursday March 12 at 10AM PST/ 11 MST/ 1PM EST in the Arctos Zoom Room. We don't have any big discussion items, if you have something you wish to discuss please put it on the agenda!
https://github.com/orgs/ArctosDB/projects/27
[Taxonomy Agenda](https://docs.google.com/document/d/1gxMD9uzsyGl2zKaMitQ_6tH33G9RUoVpwDWzuB2Xxmw/edit)

**Comment by genevieve-anderegg** [comment]
@ArctosDB/taxonomy Taxonomy committee tomorrow! Thursday May 14 at 10AM PST/ 11 MST/ 1PM EST in the Arctos Zoom Room. We don't have any big discussion items, if you have something you wish to discuss please put it on the agenda!
https://github.com/orgs/ArctosDB/projects/27
[Taxonomy Agenda](https://docs.google.com/document/d/1gxMD9uzsyGl2zKaMitQ_6tH33G9RUoVpwDWzuB2Xxmw/edit)
